int64 xCoord
int64 yCoord
sensor_msgs/Image image